src/uikit.cs

```diff
@@ -10024,15 +10030,12 @@ interface UIScreen : UITraitEnvironment {
 	}
 
 	[BaseType (typeof (UIView), Delegates=new string [] { "WeakDelegate" }, Events=new Type [] {typeof(UIScrollViewDelegate)})]
-	interface UIScrollView {
+	interface UIScrollView : UIFocusItemScrollableContainer {
 		[Export ("initWithFrame:")]
 		IntPtr Constructor (CGRect frame);
 
-		[Export ("contentOffset")]
-		CGPoint ContentOffset { get; set; }
-
 		[Export ("contentSize")]
-		CGSize ContentSize { get; set; }
+		new CGSize ContentSize { get; set; }
```

Comment:
Can this be removed as well, so that we get it from `UIFocusItemScrollableContainer` instead, or will the api diff complain?

Reply:
No, in this case the protocol just exposes the get, but not the set.